timezone/src/include/timezone/dialogs.rb

```diff
@@ -575,12 +577,7 @@ def TimezoneDialog(args)
         @ntp_used = true
         # configure NTP client
         # to prevent misusage of ntp.org we need to distinguish opensuse and SLE usage
-        base_products = Product.FindBaseProducts
-        if base_products.any? { |p| p["name"] =~ /openSUSE/i }
-          servers = (0..3).map { |i| "#{i}.opensuse.pool.ntp.org" }
-        else
-          servers = (0..3).map { |i| "#{i}.novell.pool.ntp.org" }
-        end
+        servers = Y2Network::NtpServer.default_servers.map(&:hostname)
```

Comment:
It introduces a dependency on yast2-network >= 4.2.55 isn't it?

Reply:
Maybe would be a good idea to conflict in a ntp-client version < 4.2.8 ?

https://github.com/yast/yast-ntp-client/pull/155/files#diff-2fd35b8b1087ae0be0605e2129e80172R20

Reply:
Yes, it sounds like a good plan. Thanks!